src/commands/moderation/clear.ts

```diff
@@ -0,0 +1,80 @@
+import { ApplicationCommandOptionType, ApplicationCommandType, TextChannel, Colors } from 'discord.js';
+import { slashCommand, slashCommandStructure, makeEmbed, constantsConfig, Logger } from '../../lib';
+
+const data = slashCommandStructure({
+    name: 'clear',
```

Comment:
Could rename to 'clear messages' or 'delete messages' to make the command name clearer?